src/app/components/toast/index.html

```diff
@@ -0,0 +1,92 @@
+<sky-demo-page
+  pageTitle="Toast"
+  skyuxModule="SkyToastModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The toast service launches a container over a page's content to display small messages. The <stache-code>SkyToastService</stache-code> provider launches toast messages and provides a common look-and-feel. The <stache-code>SkyToastMessage</stache-code> class watches for and triggers toast events. It also allows you to close messages.
+    </p>
+    <sky-alert
+      alertType="info">
+      The <stache-code>SkyToastService</stache-code> dynamically generates the toast component and appends it directly to the document's <stache-code>body</stache-code> element.
+    </sky-alert>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Flyout service methods">
+    <sky-demo-page-property
+      propertyName="open(config)">
+      Opens a message and displays the configured content. The <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code>. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="openMesage(message, config?)">
+      Opens a message toast and displays the provided message. The <stache-code>message</stache-code> parameter is of type <stache-code>string</stache-code> and determines the message to be displayed. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. This method returns the SkyToastMessage instance.
```

Comment:
See note above about first sentence. (Not sure my suggestion is right, but I think the similarity between this first sentence and the previous one is potentially confusing.)

In the second sentence, I don't think "string" should be a code snippet. I'd also tighten up and use active voice: "The string <stache-code>message</stache-code> parameter determines the message to display."

In the third sentence, I'd delete "additional." And in the last sentence, "SkyToastMessage" should be a code snippet.